.travis.yml

```diff
@@ -66,7 +69,11 @@ install:
   - pip install --quiet --upgrade --global-option='--without-libyaml' pyyaml
   - pip install -r requirements.txt
   - pip install -r requirements-dev.txt
-  - pip install -U ophyd --pre
+  # Install ophyd master branch if build settings indicate to do so.
+  - |
+    if [ "$OPHYD_MASTER" = true ] ; then
+      pip install git+https://github.com/bluesky/ophyd@master
```

Comment:
Does this need a `-U`?

Reply:
Apparently not (given the evidence of CI) but it wouldn't hurt to add it for robustness against future changes and for readability.